src/cleo/_utils.py

```diff
@@ -51,12 +50,12 @@ def find_similar_names(name: str, names: list[str]) -> list[str]:
     """
     Finds names similar to a given command name.
     """
-    threshold = 1e3
+    threshold = 0.4
     distance_by_name = {}
-
+    if " " in name:
+        names = [name for name in names if " " in name]
```

Comment:
Do we assume that `name` is stripped before being passed to `find_similar_names`?
Because if we don't, `find_similar_names("apple ", ["apples"])` will give us no results.

Reply:
The only places that use that function give command/namespace names already striped, so yes, that's the assumption.